### PR TITLE
Enable EIP-2384 for ice age hard fork

### PR DESCRIPTION
**Pull request by sorpaas**
There is a discussion in AllCoreDevs to include a difficulty bomb delay in Istanbul, or potentially conduct another hard fork one month after Istanbul. The proposal EIP-2384 further delays the bomb for another 4 million blocks.

Note that this is not mergable until a decision is reached.

### CODE REVIEW

File: ethcore/res/ethereum/foundation.json [hunk 1]
```diff
@@ -136,7 +136,8 @@
 				"eip100bTransition": "0x42ae50",
 				"difficultyBombDelays": {
 					"0x42ae50": "0x2dc6c0",
-					"0x6f1580": "0x1e8480"
+					"0x6f1580": "0x1e8480",
+					"0x8c6180": "0x3d0900"
```
Comment (sorpaas) [inline ethcore/res/ethereum/foundation.json]: `0x8c6180`: 9200000
`0x3d0900`: 4000000

Comment (sorpaas) [reply ethcore/res/ethereum/foundation.json]: Please review those two numbers against https://eips.ethereum.org/EIPS/eip-2384 and https://eips.ethereum.org/EIPS/eip-2387

Comment (dvdplm) [reply ethcore/res/ethereum/foundation.json]: Numbers lgtm. 
It's a bit annoying that the EIPs are drafts and 2383 is not updated with the block number yet.

Comment (niklasad1) [reply ethcore/res/ethereum/foundation.json]: https://github.com/ethereum/EIPs/pull/2408/files

**Comment by joshua-mir** [comment]
should the ropsten fork also be in this PR?

**Comment by sorpaas** [comment]
@joshua-mir Added!

**Pull request merged**